module/main.js

```diff
@@ -242,25 +264,71 @@ require('../style.css');
                     L.latLng(start.latitude, start.longitude),
                     L.latLng(destination.latitude, destination.longitude)
                 ],
-                collapsible: true,
-                geocoder: L.Control.Geocoder.nominatim(),
-                createMarker: function() { return null; } //removes the marker (we will use only pokemon icons as markers
+                collapsible:  true,
+                geocoder:     L.Control.Geocoder.nominatim(),
+                createMarker: function () { return null; } //removes the marker (we will use only pokemon icons as markers
             });
 
             route.addTo(mymap);
 
         }
 
-        function clearRoutes(){
+        function clearRoutes() {
 
-            if(route && route.removeFrom) {
+            if (route && route.removeFrom) {
 
                 route.removeFrom(mymap);
 
             }
 
         }
 
+        function filter(pokemonIds, sightingsSince, predictionsUntil) {
```

Comment:
The filter function receives only one parameter which is an object containing the three filter criteria:
```
function filter(filterOptions) {
  let pokemonIds = filterOptions.pokemonIds;
  let sightingsSince = filterOptions.sightingsSince;
  let predictionsUntil = filterOptions.predictionsUntil;
  ...
}
```